lib/rbac/access.rb

```diff
@@ -25,13 +25,14 @@ def process
 
         full_acls.each do |item|
           Rails.logger.debug("Found ACL: #{item}")
-          @acls << item if regexp.match(item.permission)
-
-          unless @admin
-            item.resource_definitions.any? do |rd|
-              @admin = true if rd.attribute_filter.key == 'id' &&
-                               rd.attribute_filter.operation == 'equal' &&
-                               rd.attribute_filter.value == '*'
+          if regexp.match(item.permission)
+            @acls << item
+            unless @admin
+              item.resource_definitions.any? do |rd|
```

Comment:
@hsong-rh can this admin check be changed to be role specific. We are working with the RBAC team to come api with an API that would allow roles to be fetched without having to check for individual accesses.
https://github.com/RedHatInsights/insights-rbac/issues/115
We can merge this for now but you might want to open a GitHub issue to convert this to be role based.
Better yet if we can extract this into a function we can just swap out the logic to role based instead of access permission based.

Reply:
@mkanoor Issue https://github.com/ManageIQ/approval-api/issues/159 is created.